csharp/Platform.Collections/IDictionaryExtensions.cs

```diff
@@ -4,17 +4,83 @@
 
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
-namespace Platform.Collections
+namespace Platform.Collections 
 {
+    /// <summary>
+    /// <para>Represents a set of extension methods <see cref="IDictionary{TKey, TValue}"/> array.</para>
+    /// <para>Представляет набор методов расширения <see cref="IDictionary{TKey, TValue}"/>.</para> 
+    /// </summary>  
     public static class IDictionaryExtensions
     {
+        /// <summary>    
+        /// <para> 
+        /// Returns the value associated with the passed parameters  
+        /// <paramref name="key"/> in <paramref name="dictionary"/> 
+        /// </para>  
+        /// <para> 
+        /// Возвращает значение связанное с  
+        /// <paramref name="key"/> в  
+        /// <paramref name="dictionary"/> или значение по умолчанию, если оно не существует.
+        /// </para> 
+        /// </summary>   
+        /// <paramref name="dictionary"> 
+        /// <para>  
+        /// <see cref="IDictionary<TKey, TValue>"/> from which to find the value associated with the <paramref name="key"/>.
+        /// </para> 
+        /// <see cref="IDictionary<TKey, TValue>"/> из которого нужно найти значение, связанное с <paramref name="key"/>.  
+        /// <paramref name="key"> 
+        /// <para>The key.</para> 
+        /// <para>Ключ.</para>  
+        /// <returns> 
+        /// <para> 
+        /// The value associated with the   
+        /// <paramref name="key"/> in the <paramref name="dictionary"/> 
+        /// </para>  
+        /// <para> 
+        /// Значение, связанное с
+        /// <paramref name="key" в <paramref name="dictionary"/>
+        /// </para> 
+        /// </returns>        
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static TValue GetOrDefault<TKey, TValue>(this IDictionary<TKey, TValue> dictionary, TKey key)
         {
             dictionary.TryGetValue(key, out TValue value);
             return value;
         }
 
+        /// <summary>    
+        /// <para>Gets the value associated with the  
+        /// <paramref name="key" in the  
```

Comment:
You have done opposite actions I have asked. Do not break lines. Write the text inside `para` tags without new lines, on one line.